docs/Next-Step-Server-REST-API-Reference.md

```diff
@@ -1441,8 +1442,61 @@ Alternative with `POST` method for environments which do not allow `PUT` methods
 
 ```json
 {
-  "status" : "OK",
-  "responseObject" : null
+  "status" : "OK"
+}
+```
+
+### Update user for an operation
+
+Updates user ID and organization ID for an operation.
+
+<table>
+    <tr>
+        <td>Method</td>
+        <td><code>PUT</code></td>
+    </tr>
+    <tr>
+        <td>Resource URI</td>
+        <td><code>/operation/user</code></td>
+    </tr>
+</table>
+
+Alternative with `POST` method for environments which do not allow `PUT` methods:
+<table>
+    <tr>
+        <td>Method</td>
+        <td><code>POST</code></td>
+    </tr>
+    <tr>
+        <td>Resource URI</td>
+        <td><code>/operation/user/update</code></td>
+    </tr>
+</table>
+
+
+#### Request
+
+- Headers:
+    - `Content-Type: application/json`
+
+```json
+{
+  "requestObject": {
+    "operationId": "0a044408-aea0-433a-80cf-6371dc2a76c0",
+    "userId": "roman",
```

Comment:
nitpick:  😄 Can we use our real names in doc or should we stay with some anonymous things? e.g. `userId`, `john`

Reply:
I already started switching to anonymous users in Data Adapter documentation, will switch here, too.

Reply:
I will do final documentation update by the end of week, there are multiple issues in Next Step documentation because we are changing the REST API a lot.